### PR TITLE
Page and  Header component #165

### LINKED ISSUE #165: Page and Header Component
Create the Page and Header Component in Lakefront. Write stories to demonstrate the functionality and unit test cases.

### PR DESCRIPTION
**Pull request by NehalBhandariSlalom**
This PR creates `Page` and `Header` Component. If approved, this will close #165 

1. Page Component
2. Header Component

Screen Shot 2021-09-15 at 1 13 16 PM
Screen Shot 2021-09-15 at 2 27 20 PM

### CODE REVIEW

**Review by jason-ransom-slalom — CHANGES_REQUESTED** [review]
Need to add additional styles for the header components.

```json
color: theme.colors.storm,
fontSize: 30,
fontWeight: 600,
marginBottom: 0
```

File: src/Page/Page.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    className?: string;
+}
+
+/**
+ * Page Component
+ * 
+ * The Page Component can be used to render other child components.
+ */
+
+const Page: FC<PageProps> = props =>
```
Comment (jason-ransom-slalom) [inline src/Page/Page.tsx:19]: The indentation in this file seems different than other component files. Can you update to match?

File: src/Page/Header.tsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+import { ThemeProvider } from '@emotion/react';
+import { StyledHeader } from './pageStyles';
+import theme from 'src/styles/theme';
+
+ /**
+ * Header Component
+ * 
+ * The Header component is used to render the header.
+ */
+const Header: FC = props =>
```
Comment (jason-ransom-slalom) [inline src/Page/Header.tsx:11]: The indentation in this file seems different than other component files. Can you update to match?

File: src/Page/Page.tsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+{
+    const { children, className } = props;
```
Comment (jason-ransom-slalom) [inline src/Page/Page.tsx:21]: Also, please destructure `children` and `className` in the function call as opposed to in the body of the function as we do in other components.

File: src/Page/Header.tsx [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+{
+    const { children } = props;
```
Comment (jason-ransom-slalom) [inline src/Page/Header.tsx:13]: Also, please destructure  `children` in the function call as opposed to in the body of the function as we do in other components.

**Review by jason-ransom-slalom — CHANGES_REQUESTED** [review]

File: src/Page/Page.tsx (hunk 1 above)
Comment (NehalBhandariSlalom) [reply src/Page/Page.tsx]: Done

**Comment by NehalBhandariSlalom** [comment]
> Need to add additional styles for the header components.
> 
> ```json
> color: theme.colors.storm,
> fontSize: 30,
> fontWeight: 600,
> marginBottom: 0
> ```

Added the styles.

File: src/Page/Page.tsx (hunk 3 above)
Comment (NehalBhandariSlalom) [reply src/Page/Page.tsx]: done

File: src/Page/Header.tsx (hunk 4 above)
Comment (NehalBhandariSlalom) [reply src/Page/Header.tsx]: done

**Review by jason-ransom-slalom — APPROVED** [review]

**Review by adorack-tri — APPROVED** [review]

**Review by amirkibbar — COMMENTED** [review]
lgtm

**Pull request merged**